### ISSUE TITLE
Not working under sudokuhax on 3ds

### ISSUE THREAD

**Issue by Drenn1** [devkitPro/nds-hb-menu#1]
I've heard that hbmenu works at least for some dsi users, so it's strange that it's not working on my 3ds after installing sudokuhax. When I try to boot anything I just get a white screen. Both the v0.4.5 release and a newly compiled version have this issue. The apps do work if used directly as boot.nds.

**Comment by ApacheThunder** [comment]
It appears to be related to cluster size. if you have a 32GB (this could effect any capacity card though) formatted to anything beyond 8192K cluster size. hbMenu will fail to boot anything. (white screen). However I would like to see the issue fixed as actually using that cluster size causes some bugs with AGB_FIRM. (screen framebuffer bug of some kind?). So I would like to not have to alter cluster size to get this to work.

**Comment by WinterMute** [comment]
It's not related to cluster size at all, I found a 16GB card where it fails but it's working fine on 8GB cards with 8K clusters.

I've updated the sdmmc code in libnds and the hbmenu bootloader - it works with all of the cards I can dig up >= 8GB. Not sure where you got a version that isn't SDHC compatible, i don't think this has ever been tested on non SDHC cards with dsiwarehax.

Try 0.4.7 from https://sourceforge.net/projects/devkitpro/files/hbmenu/
